odoo/addons/base/tests/test_ir_mail_server.py

```diff
@@ -217,7 +217,7 @@ def test_mail_server_priorities(self):
                 self.assertEqual(mail_server, expected_mail_server)
                 self.assertEqual(mail_from, expected_email_from)
 
-    @mute_logger('odoo.models.unlink')
+    @mute_logger('odoo.models.unlink', 'odoo.addons.base.models.ir_mail_server')
     def test_mail_server_send_email(self):
```

Comment:
Also there is a 'test_mail_server_send_email' test method in 'mail' module, using domains and stuff like that, maybe it deserves its own additional test ?